app.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./servers.controllers");
+
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+app.use((err, req, res, next) => {
+  if (err.code === "22P02") {
```

Comment:
Only have the error handling in that that is relevant to the task.  You are not able to make a bad request.   Avoid pasting in boilerplate code.